CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
-## 0.84.1 / 2025-08-06
+## 0.85.0 / 2025-08-21
 
 * [CHANGE/BUGFIX] Add the `--watch-referenced-objects-in-all-namespaces` CLI argument. When enabled, the operator watches for secrets and configmaps in both workload and configuration resources. It ensures that reconciliation happens when a referenced secret/configmap is updated. #7615
+* [FEATURE] Add `resendDelay` field to the ThanosRuler CRD. #7753
```

Comment:
we can also include https://github.com/prometheus-operator/prometheus-operator/pull/7669, https://github.com/prometheus-operator/prometheus-operator/pull/7766?

Reply:
Ah yeah, I missed those!